tests/build_stitched_index.cpp

```diff
@@ -0,0 +1,811 @@
+// author: Siddharth Gollapudi
+// email: t-gollapudis@microsoft.com
+
+#include <boost/program_options.hpp>
+#include <bits/types/struct_iovec.h>
```

Comment:
Can we replace this std library equivalents? Windows build fails here.

Reply:
Neel and I did some work on designing an STL algorithm, and they all had their downsides. I would like to keep this implementation due to performance reasons when running on Linux, but I can include an alternative algorithm for Windows platforms?